lab-Jp/gulpfile.js

```diff
@@ -0,0 +1,51 @@
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+
+'use strict';
+
+var testFiles = ['../test/cowsay_test.js'];
```

Comment:
This is referencing the gulpfile.js in the lab-Jp folder in lab-01 project folder.  The instructions say it should be global but I was told to put it in the lab-Jp folder.